packages/flutter/lib/src/widgets/two_dimensional_scroll_view.dart

```diff
@@ -109,7 +110,10 @@ abstract class TwoDimensionalScrollView extends StatelessWidget {
   final DragStartBehavior dragStartBehavior;
 
   /// {@macro flutter.widgets.scroll_view.keyboardDismissBehavior}
-  final ScrollViewKeyboardDismissBehavior keyboardDismissBehavior;
+  ///
+  /// If [keyboardDismissBehavior] is null then it will fallback to the closest
+  /// [ScrollConfiguration] provided by the context.
```

Comment:
```suggestion
  /// If [keyboardDismissBehavior] is null then it will fallback to the inherited
  /// [ScrollBehavior.keyboardDismissBehavior].
```